compiler/z/codegen/BinaryEvaluator.cpp

```diff
@@ -901,7 +907,8 @@ lDivRemGenericEvaluator64(TR::Node * node, TR::CodeGenerator * cg, bool isDivisi
    dependencies->addPostCondition(remRegister, TR::RealRegister::LegalEvenOfPair);
    dependencies->addPostCondition(quoRegister, TR::RealRegister::LegalOddOfPair);
 
-   if (!doConditionalRemainder &&
+   // TODO: Remove this Java-ism from OMR and push it into OpenJ9
```

Comment:
If this is covered by #3685 then why not mention the issue here rather than a TODO?

Reply:
Ah yes, the issue was opened after-the-fact. Totally forgot to go back and update the reference in the code. Fixed in 7e888b5